docs/examples/object/robot.py

```diff
@@ -0,0 +1,45 @@
+from compas_robots import RobotModel
+from compas_robots.resources import GithubPackageMeshLoader
+
+from compas_viewer import Viewer
+from compas_viewer.layout import Slider
+from compas_viewer.layout import Treeform
+from compas_viewer.scene.robotobject import RobotModelObject
+
+viewer = Viewer(rendermode="lighted")
+
+
+github = GithubPackageMeshLoader("ros-industrial/abb", "abb_irb6600_support", "kinetic-devel")
+model = RobotModel.from_urdf_file(github.load_urdf("irb6640.urdf"))
+model.load_geometry(github)
+
+
+robot_object: RobotModelObject = viewer.add(model, show_lines=False, configuration=model.random_configuration())  # type: ignore
```

Comment:
Robot Object can set show_lines to False by default

Reply:
it indeed looks nicer without the lines... but users might want to see the lines for some reason... i would align it to the configuration settings.